src/issuer/pre-signup/merchant/app.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+from aws_lambda_powertools.utilities.typing import LambdaContext
+from aws_lambda_powertools.logging import correlation_paths
+from aws_lambda_powertools import Logger
+from aws_lambda_powertools import Tracer
+from aws_lambda_powertools import Metrics
+from aws_lambda_powertools.metrics import MetricUnit
+from pyqldb.driver.qldb_driver import QldbDriver
+from pyqldb.config.retry_config import RetryConfig
+from qldb_helper import QLDBHelper
+
+tracer = Tracer()
+logger = Logger()
+metrics = Metrics(namespace="Powertools")
+
+
+@tracer.capture_method
+def signup_confirmation(event: dict, context: LambdaContext):
```

Comment:
Is event a consistent schema? If so it's probably better to have a more strongly typed class or object